packages/connect-popup/e2e/tests/methods.test.ts

```diff
@@ -23,13 +28,28 @@ test.beforeAll(async () => {
     await TrezorUserEnvLink.connect();
 });
 
+test.afterEach(async () => {
+    if (context) {
+        // BrowserContext has to start fresh each test.
+        // https://playwright.dev/docs/api/class-browsercontext#browser-context-close
+        await context.close();
+        await new Promise(resolve => setTimeout(resolve, 1000));
+    }
+});
+
 test.afterAll(() => {
     buildOverview({ emuScreenshots });
 });
 
-let device = {};
-
-fixtures.forEach(f => {
+// TODO: for now those methods are not working in connect-explorer in webextension env so we skip them.
+const methodsUrlToSkipInWebExtension = ['composeTransaction', 'wipeDevice', 'recoverDevice'];
```

Comment:
what happens when you try them manually without playwright in connect-explorer in webextension?

Reply:
The same, infinite loading, and error in console. I am quite sure it is some kind of parsing error in connect-explorer, but did not investigate yet. I would like to investigate it , it should not be a bit deal.

Reply:
yes, please do, does log.html help here?